docs/version0.49.md

```diff
@@ -37,6 +38,12 @@ macOS 12 is out of support and is removed from the list of supported platforms.
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
+### Change to the shared classes cache generation number
+
+The shared classes cache generation number is incremented because of a change in the format of ROMClasses that are stored in the shared classes cache. A new flag `J9AccClassIsShared` is added to ROMClasses to indicate whether a ROMClass was loaded from a shared classes cache or from a VM. The increment in the shared classes cache generation number causes the VM to create a new shared classes cache, rather than re-creating or reusing an existing cache.
+
+To save space, all existing shared caches can be removed unless they are in use by an earlier release. For more information, see [Housekeeping](shrc.md#housekeeping) and [`-Xshareclasses`](xshareclasses.md).
```

Comment:
I want to see this moved up, and the part starting with "A new flag `J9AccClassIsShared`" moved afterwards.

Reply:
The shared classes cache generation number is incremented. The increment in the shared classes cache generation number causes the VM to create a new shared classes cache, rather than re-creating or reusing an existing cache.

To save space, all existing shared caches can be removed unless they are in use by an earlier release. For more information, see [Housekeeping](shrc.md#housekeeping) and [`-Xshareclasses`](xshareclasses.md).

The shared classes cache generation number is modified because of a change in the format of ROMClasses that are stored in the shared classes cache. A new flag `J9AccClassIsShared` is added to ROMClasses to indicate whether a ROMClass was loaded from a shared classes cache or from a VM

Reply:
The flow seemed better this way whereby we have first specified what the change is (number incremented because of a format change) and then the reason for that format change (the new flag added) and then the result of the change (creation of a new cache). After we have specified the creation of a new cache, we have moved to explain how we can save space (removing them with -Xshareclasses).

Moving the information related to the flag last breaks the flow. If it is not resulting in a technical inaccuracy, I would recommend retaining the specified flow.

Reply:
People don't read things to completion, the important things should be first.

Reply:
I have incorporated the suggested changes. Please check and confirm. Thanks!